src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -310,12 +310,9 @@ private static final class Suppression {
         /** The regexp which is used to match the event ID.*/
         private final Pattern eventIdRegexp;
 
-        /** Suppression text.*/
-        private final String text;
         /** Suppression line.*/
         private final int lineNo;
-        /** Suppression column number.*/
-        private final int columnNo;
```

Comment:
I don't care about the mutation, but this will flaw the `Suppression` class and might introduce bugs in the future.  Two mutations on the **same line** in **different columns** can now be **possibly the same**. 

@romani @rdiachenko Please share your opinion on this.

Reply:
https://checkstyle.org/filters/suppresswithplaintextcommentfilter.html
Suppression is a line comment, usually.

In our examples we never even tried to use suppression on/off comments to be granulated to columns.
We just try match a line where OFF comment and ON comment, all violations between will be suppressed.

We do use only lines:
```
private boolean isInScopeOfSuppression(AuditEvent event) {
            return lineNo <= event.getLine();
        }
```
https://github.com/checkstyle/checkstyle/blob/223ed3981ebf088c005dc7c8bf2d075194f77ae1/src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java#L433